### ISSUE TITLE
Mouse sometimes get stuck in a "blocked" state

### ISSUE THREAD

**Issue by olejorgenb** [paperwm/PaperWM#80]
Root cause: Clicking on a hidden window actor blocks all subsequent mouse input. (Probably inside the window's input area)

Might be X11 specific.

Simplest to reproduce by slowing down animations: `imports.gi.St.set_slow_down_factor(3)`
Turning on clone marks make it simpler to see what's going on: `Extension.imports.utils.toggleCloneMarks()`

Trigger an animation and click on a non-selected window before the animation is done.

(Simplest to super drag a window and click on another window when done (move-done is not triggered on grabEnd so there's some time))

The focused window have keyboard focus, but the mouse is completely blocked (panel too). It can be moved but clicks are blocked.

Activating the navigator escapes from the blocked state. Changing focus by some other means (eg. rofi) does *not* break the spell.

**Comment by hedning** [comment]
I'm unable to reproduce in Wayland, I would hazard a guess that this is the problem: https://github.com/paperwm/PaperWM/blob/14ab95a6f50d2957207b61e0bf35e1d7d7f9c4e1/tiling.js#L148

Thought it was safe, but I didn't test that thoroughly.

**Comment by olejorgenb** [comment]
I tried turning it off, but now that I think about it I only did it when I was in the "locked" state.

**Comment by olejorgenb** [comment]
Yep, turning it off prevents entering the state. But then we get that pesky popupmenu instead..

**Comment by olejorgenb** [comment]
When I connect to "button-press-event" on the background, trigger the bug and click around, the signal handler is not called until I "unlock" (by using the navigator).

I also forgot to note that the bug is not triggered when clicking directly on the background.
